modules/srv6/gr_srv6.h

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
If this is a non exported header, can you rename it to `srv6.h` without the `gr_` prefix and include it with double quotes?